### PR TITLE
GULAppDelegateSwizzler - comments updated

### PR DESCRIPTION
**Pull request by maksymmalyhin**

### CODE REVIEW

**Comment by maksymmalyhin** [comment]
I am cancelling the rest of Travis jobs because there are no actual code changes.

**Comment by maksymmalyhin** [comment]
I would say `delegate == nil` is the only common case way, but having two more methods (potentially more added?) in between made me think that validating the name itself is a good idea.

File: GoogleUtilities/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- Update comments in `GULAppDelegateSwizzler` (#4518)
```
Comment (wilhuff) [inline GoogleUtilities/CHANGELOG.md]: Suggestion: these comment changes don't affect consumers of this package so don't bother your users with changelog entries. If users want to see everything we do they can browse git history. This should be a quick list of things that might affect them or might prove useful.

Comment (maksymmalyhin) [reply GoogleUtilities/CHANGELOG.md]: Good point, I'll remove it then.

**Pull request merged**